pyvista/plotting/plotting.py

```diff
@@ -1807,6 +1804,443 @@ def update(self, stime=1, force_redraw=True):
         if force_redraw:
             self.render()
 
+    def add_composite(
+        self,
+        dataset,
+        color=None,
+        style=None,
+        scalars=None,
+        clim=None,
+        show_edges=None,
+        edge_color=None,
+        point_size=5.0,
+        line_width=None,
+        opacity=1.0,
+        flip_scalars=False,
+        lighting=None,
+        n_colors=256,
+        interpolate_before_map=True,
+        cmap=None,
+        label=None,
+        reset_camera=None,
+        scalar_bar_args=None,
+        show_scalar_bar=None,
+        multi_colors=False,
+        name=None,
+        render_points_as_spheres=None,
+        render_lines_as_tubes=False,
+        smooth_shading=None,
+        split_sharp_edges=None,
+        ambient=0.0,
+        diffuse=1.0,
+        specular=0.0,
+        specular_power=100.0,
+        nan_color=None,
+        nan_opacity=1.0,
+        culling=None,
+        rgb=None,
+        categories=None,
+        below_color=None,
+        above_color=None,
+        annotations=None,
+        pickable=True,
+        preference="point",
+        log_scale=False,
+        pbr=False,
+        metallic=0.0,
+        roughness=0.5,
+        render=True,
+        component=None,
+        color_missing_with_nan=False,
+        **kwargs,
+    ):
+        """Add a composite dataset to the plotter.
+
+        Parameters
+        ----------
```

Comment:
I called out a few examples, but many parameters are missing defaults.  I assume this also means the same for `add_*`

Reply:
I'm going to leave the rest of the `add_*` alone, but I'll fix this docstring.